### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by wangping1408** [TheSouthFrog/stylealign#5]
Hi!
Your project is great. I have some questions about the dataset section.  In your paper, used the AFLW dataset. The original number of the dataset is 19,  How to label it as 68 label points? and the WFLW is 98 points. Are 68 points used in actual training? How is it obtained? I am confused about the processing of the data set, I hope you can give me some help. Thank you very much!

**Comment by TheSouthFrog** [comment]
As stated in the paper, AFLW-68 follows 300W's 68pts annotation. We annotate it in order to make the cross-dataset comparison between AFLW and 300W dataset available. For WFLW dataset, we just use 98 points' gaussian map as the input, it is just a pre-processing data loading step. You can refer to the dataloader.py for the details. 

Note that in order to allow fair comparison, a generator on each dataset is trained solely on this dataset's training dataset, e.g. 68pt for 300W experiment, 98 pt for WFLW experiment.

**Comment by wangping1408** [comment]
Thank you , for your answer to my question.
Can you tell me how do you mark the AFLW-68 data as 68 points? Is it manually labeled or an algorithmically labeled?

**Comment by TheSouthFrog** [comment]
They are manually labeled, following 300W annotation rule.

**Comment by wangping1408** [comment]
Can you provide the annotated AFLW-68 datasets? I want to use your datasets set to train my own model.




------------------ 原始邮件 ------------------
发件人: "TheSouthFrog"<notifications@github.com>; 
发送时间: 2019年9月30日(星期一) 下午5:51
收件人: "TheSouthFrog/stylealign"<stylealign@noreply.github.com>; 
抄送: "寻找那辆单车"<1262362093@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [TheSouthFrog/stylealign] datasets (#5)




They are manually labeled, following 300W annotation rule.
